tests/unit/test_utils.py

```diff
@@ -37,10 +37,13 @@ def test_read_pandas_csv():
 
 def test_read_pandas_json():
     p = Path("dataframe.json")
-    with pytest.raises(ValueError) as e:
+    with pytest.raises((ValueError, FileNotFoundError)) as e:
```

Comment:
Split into two tests depending on the pandas version (one could be dropped eventually)

Reply:
This would again require a version check (which haven't been used in this library so far)... I don't think it's worth the added test complexity.